msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -21,6 +21,54 @@ The Azure DevOps message extension uses link unfurling to look for URLs pasted i
 
 :::image type="content" source="~/assets/images/compose-extensions/messagingextensions_linkunfurling.png" alt-text="Example of link unfurling":::
 
+## App less adaptive card preview with schema.org templates
+
+Schema.org is a collaborative, community based activity with a mission to create, maintain, and promote schemas for the structured data on the internet, web pages, email and beyond.
+
+### How does it work
+
+When metadata of the webpage is tagged to a type of product it unfurls to an adaptive card. App type plays an crucial role, each type will have a designated template. The adaptive card is upgraded with lot more action buttons
+
+:::image type="content" source="../../assets/images/messaging-extension/meta.png" alt-text="Meta data"border="true"lightbox="../../assets/images/messaging-extension/meta-data.png":::
+
+### Adding meta data to the website
+
+Add structured schema.org JSON-LD metadata to the website. Few of the templates with @types:
+
+Creative works: [CreativeWork](https://schema.org/CreativeWork), [Book](https://schema.org/Book), [Movie](https://schema.org/Movie), [MusicRecording](https://schema.org/MusicRecording), [Recipe](https://schema.org/Recipe) and [TVSeries](https://schema.org/TVSeries).
+
+Embedded non-text objects: [AudioObject](https://schema.org/AudioObject), [ImageObject](https://schema.org/ImageObject) and [VideoObject](https://schema.org/VideoObject)​
+
+[Event​](https://schema.org/Event)
+
+[Health and medical types](https://schema.org/docs/meddocs.html): notes on the health and medical types under [MedicalEntity](https://schema.org/MedicalEntity).​
+
+[Organization​](https://schema.org/Organization)
+
+[Person​](https://schema.org/Person)
+
+[Place](https://schema.org/Place)
+
+[LocalBusiness](https://schema.org/LocalBusiness)
+
+[Restaurant](https://schema.org/Restaurant)
+
+[Product](https://schema.org/Product), [Offer](https://schema.org/Offer), [AggregateOffer​](https://schema.org/AggregateOffer)
+
+[Review](https://schema.org/Review), [AggregateRating​](https://schema.org/AggregateRating)
+
+[Action](https://schema.org/Action)
+
+Following is an instance of adding structured schema.org JSON-LD metadata to the website.
+
+:::image type="content" source="../../assets/images/messaging-extension/metadata-1.png" alt-text="structured schema.org"border="true":::
```

Comment:
This is a screenshot of metadata, but it doesn't actually show how to add it to a webpage.

Reply:
removed screenshot.